docs/1.13/03-Tutorials2/06-Deploy-Prisma-Servers/07-Dokku.md

```diff
@@ -0,0 +1,136 @@
+---
+alias: mohj5eiwot
```

Comment:
this alias is already used by the `Zeit-Now-and-Google-Cloud-SQL` tutorial, duplicate aliases are not allowed in our docs as an alias is a globally unique identifier for a page. you can use this one instead: `zie0ahhaox`